src/components/wrapper/wrapper.js

```diff
@@ -45,7 +45,7 @@ export const Wrapper = ({ displayGlobalToolbar, theme }) => {
     >
       <h1 className="pipeline-title">Kedro-Viz</h1>
       <Router>
-        {displayGlobalToolbar ? (
+        {displayGlobalNavigation ? (
           <GraphQLProvider>
             <GlobalToolbar isOutdated={isOutdated} />
```

Comment:
are we keeping the same name for the component ?

Reply:
Yes, keeping the same name for the component.